src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -103,6 +106,12 @@ public JsonAdaptedPerson(Person source) {
         priority = source.getPriority().value;
         assert priority != null : "Value stored in priority should not be null";
 
+        if (source.getPolicy() != null) {
+            policy = new JsonAdaptedPolicy(source.getPolicy());
+            assert policy != null : "Value stored in address should not be null";
```

Comment:
Perhaps the assertion can print "Policy should not be null"
Perhaps we should check policy.getPolicyName() != null and policy.getPolicyDescription() != null as well
and print "Policy name should not be null" and "Policy description should not be null"